doc/prefacios/prefacio.tex

```diff
@@ -1,68 +1,66 @@
 \thispagestyle{empty}
 
 \begin{center}
-{\large\bfseries Título \\ Subtítulo }\\
+{\large\bfseries Flixbuff \\ Una red social para seriéfilos }\\
 \end{center}
 \begin{center}
-Nombre Del Estudiante\\
+Víctor Cabrita Gómez\\
 \end{center}
 
-%\vspace{0.7cm}
-
-\vspace{0.5cm}
-\noindent{\textbf{Palabras clave}: \textit{software libre}
 \vspace{0.7cm}
 
-\noindent{\textbf{Resumen}\\
-	
+\noindent\textbf{Resumen}\\
+
+Con la irrupción y popularidad de diferentes servicios de streaming como Netflix, HBO, etc. el catálogo de series que uno puede ver ya no se limita a las que echen por la televisión. En este vasto mar de opciones, todos tenemos la misma pregunta: ``¿Qué debería ver?''. A pesar de existir infinidad de sitios web que clasifican las series de moda en un ranking, lo que a uno más le interesa es la opinión de su círculo cercano.\\
 
-\cleardoublepage
+Flixbuff es la aplicación ideal para aquellos que quieren saber qué opinan sus amigos sobre las series del momento. Es una red social en la que los usuarios pueden escribir su opinión sobre las series que hayan visto y leer la opinión de los usuarios que siguen sobre las mismas.
+
+\cleardoublepage{}
 
 \begin{center}
-	{\large\bfseries Same, but in English}\\
+	{\large\bfseries Flixbuff \\ A social network for TV Series enthusiasts}\\
 \end{center}
 \begin{center}
-	Student's name\\
+	Víctor Cabrita Gómez\\
 \end{center}
-\vspace{0.5cm}
-\noindent{\textbf{Keywords}: \textit{open source}, \textit{floss}
 \vspace{0.7cm}
 
-\noindent{\textbf{Abstract}\\
+\noindent\textbf{Abstract}\\
 
+With the increase in popularity of different streaming services such as Netflix, HBO, etc. the catalog of TV series that one can see already no longer limits itself to those that broadcast on television. In this vast market of options, everyone has the same question: ``What should I watch?'' Despite the fact that there are many websites that classify TV series in a ranking, what one more interested is the opinion of their friends.\\

```

Comment:
"Such as"  does preclude "etc". Just say "such as Netflix or HBO" (BTW, it's called HBO max now)

Reply:
done